main.go

```diff
@@ -647,6 +648,26 @@ func loadConfig(filename string, queryAWSUserData bool, queryGCPMetaData bool) (
 	return c, nil
 }
 
+var exposer expose.Exposer
```

Comment:
Should we have an exposer per feature that exposes?

Reply:
No, because they would need different websocktunnel client IDs, and I'm hoping to just use the workerGroup/workerId for that.

Reply:
ok